kaitai_utils.py

```diff
@@ -101,19 +129,33 @@ def _object_to_dict(self, instance: Any, path: str) -> Generator[Dict[str, Any],
             elif _is_list(value_object):
                 yield _to_lower_camel_case(key), self._list_to_dict(value_object, path)
             elif isinstance(value_object, bytes):
-                yield self.process_byte_array(value_object, path, key)
+                self._process_bytes(value_object, offsets, path, key)
             else:
                 yield _to_lower_camel_case(key), _process_value(value_object)
 
-    def process_byte_array(self, value_object: bytes, path: str, key: str):
+    def _process_bytes(self, value_object, offsets, path, key):
```

Comment:
Can you add type-hinting here? 
E.g. to the reader it can be more clear that offsets can be `None`.

Did you check the code with mypy?

Reply:
(when i look at the code in this method, offsets can not be None?) But it _can_ be None when called from `_list_to_dict` ?